campaignion_email_to_target/campaignion_email_to_target.module

```diff
@@ -169,8 +171,8 @@ function campaignion_email_to_target_tokens($type, array $tokens, array $data =
     $submission = new Submission($node, $s);
     $replacements = Tokens::submissionTokens($tokens, $submission);
   }
-  if ($type == 'email-to-target' && !empty($data['email-to-target'])) {
-    $replacements = Tokens::messageTokens($tokens, $data['email-to-target']);
+  if ($type == ('target' || 'email-to-target') && !empty($data['target'])) {
```

Comment:
This condition won’t work. It will only at the new tokens because `('target' || 'email-to-target') == 'target'`.